### ISSUE TITLE
Storage mismatch @mainnet's block 3960417

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/neo-go#966]
file BlockStorage_4000000/dump-block-3961000.json: block 3960417, changes number mismatch: 24 vs 26

Missing items:
```
         {
            "value" : "00b5820500056f776e6572001427868d17ee815d4f9741d145f6a2413eab01a8ba000363696400146b367435423535717a72755172664f734b6850640005737461746500044f50454e000b7375626d697373696f6e738001004c537b553d27868d17ee815d4f9741d145f6a2413eab01a8ba264b3d437b4f3d27868d17ee815d4f9741d145f6a2413eab01a8ba26493d6b367435423535717a72755172664f734b6850647d7d000974696d657374616d700204fc3a1e5d00",
            "key" : "7bce75549799b997a93dad0de4b5247bd4a477d2437b4f3d27868d17ee815d4f9741d14500f6a2413eab01a8ba26493d6b36743542003535717a72755172664f734b6850647d000000000000000000000000000000000010",
            "state" : "Changed"
         },
         {
            "value" : "00fa820c000a6368616c6c656e676572001427868d17ee815d4f9741d145f6a2413eab01a8ba000d6368616c6c656e67655f6b65790030437b4f3d27868d17ee815d4f9741d145f6a2413eab01a8ba26493d6b367435423535717a72755172664f734b6850647d0006766f746572738000000a646966666572656e636500000009617070726f766572738000000e617070726f7665725f636f756e740000000972656a6563746572738000000e72656a65637465725f636f756e74000000067374617475730008415050524f564544000573746174650006434c4f534544000974696d657374616d700204fb3b1e5d0007636c61696d656400024e4f00",
            "state" : "Added",
            "key" : "7bce75549799b997a93dad0de4b5247bd4a477d2537b553d27868d17ee815d4f9741d14500f6a2413eab01a8ba264b3d437b4f3d2700868d17ee815d4f9741d145f6a2413eab0001a8ba26493d6b367435423535717a7200755172664f734b6850647d7d0000000004"
         }
```

Failing transaction:
```json
{
   "id" : 1,
   "result" : {
      "txid" : "0x71c43cc32ae5336622d7105d1c96387c8cca6c948beac29edfef46712ba5ffeb",
      "executions" : [
         {
            "stack" : [],
            "vmstate" : "FAULT",
            "notifications" : [],
            "contract" : "0xd6e05eab0f0fe987481b59aa804fee6f62f660a5",
            "trigger" : "Application",
            "gas_consumed" : "1.714"
         }
      ]
   },
   "jsonrpc" : "2.0"
}
```
C# log:
```json
{
   "jsonrpc" : "2.0",
   "id" : 1,
   "result" : {
      "txid" : "0x71c43cc32ae5336622d7105d1c96387c8cca6c948beac29edfef46712ba5ffeb",
      "executions" : [
         {
            "trigger" : "Application",
            "notifications" : [],
            "vmstate" : "HALT",
            "gas_consumed" : "3.867",
            "stack" : [
               {
                  "type" : "Integer",
                  "value" : "1"
               },
               {
                  "value" : "1",
                  "type" : "Integer"
               },
               {
                  "value" : "537b553d27868d17ee815d4f9741d145f6a2413eab01a8ba264b3d437b4f3d27868d17ee815d4f9741d145f6a2413eab01a8ba26493d6b367435423535717a72755172664f734b6850647d7d",
                  "type" : "ByteArray"
               }
            ],
            "contract" : "0xd6e05eab0f0fe987481b59aa804fee6f62f660a5"
         }
      ]
   }
}
```

**Comment by roman-khimov** [comment]
It's from master-2.x + #930. @fyrchik, can you take a look at it?

**Comment by fyrchik** [comment]
Closed via #974.
